### ISSUE TITLE
Bug:public post boosted to "followers only" cannot be re-boosted, even though now seemingly public.

### ISSUE THREAD

**Issue by paul4565** [Dimillian/IceCubesApp#1180]
## Environment:

 - OS: 16.3.1
 
 - IceCubesApp version: 1.6.1
 

## Description

Hi Thomas.. although since your fix in v1.6.1 the boosted post now shows as public instead of followers only,the post is still not boostable from the Home timeline, but only from the original post. (Boost icon not clickable and boost option not available when swiping. Sorry to be fussy! 

## Related Issues

 - [x] Search that this bugs don't already exist before creating it.
 

[#1096]()

**Comment by Dimillian** [comment]
It's consistent with the web: 
<img width="581" alt="image" src="https://user-images.githubusercontent.com/535509/223338702-6889ef59-25e9-4c1c-90d7-865ecc639184.png">
 You can't boost a post that is follower only?

**Comment by paul4565** [comment]
Agreed, but if the screenshot had been a public post boosted to followers only rather than an original followers-only post then shouldn't it be boostable from there (because the original is public)?

Ive boosted your "50 shades of purple" post from yesterday to followers only (from another app as ice cubes doesn't do this). If using your test account you can navigate to it and find where I've boosted it, then I think you might see that you can't re-boost it from there but only by going to the original public post (ok one extra click... I'm lazy!). 😐

**Comment by paul4565** [comment]
Have now discovered that as well as the boosting issue, the same boosted post can't be **replied to** publicly (but reply will be to followers only) unless clicking through to the original public post. More work for you!
